content/pages/98_meetings_fr.md

```diff
@@ -12,6 +12,10 @@ Status: hidden
 
 Ici se trouve une liste des meetings du groupe. Chaque page regroupe les décisions / remarques prises.
 
+### 2021
+
+* [Online, 11.03.2020](meeting_minutes/2020-03-11)
```

Comment:
```suggestion
* [Online, 11.03.2021](meeting_minutes/2021-03-11)
```

Reply:
Hello!

Yes, as soon as it is done... 
I'm a bit late for the PV, I'll try to finish it for tomorrow.
Thanks !